Pod/Classes/URBNAlertView.m

```diff
@@ -111,12 +119,14 @@ - (instancetype)initWithAlertConfig:(URBNAlertConfig *)config alertStyler:(URBNA
         NSDictionary *metrics = @{@"sectionMargin" : self.alertStyler.sectionVerticalMargin,
                                   @"btnH" : self.alertStyler.buttonHeight,
                                   @"lblHMargin" : self.alertStyler.labelHorizontalMargin,
-                                  @"titleVMargin" : titleMargin,
+                                  @"topVMargin" : @([titleMargin doubleValue] + [self.alertStyler.topMargin doubleValue]),
```

Comment:
Is there any news? Just wondering shall we wait to use this in our project with your CocoaPod or start to create a new pod with this functionality? I don't push you, but if you've got any doubts just let me know - we will do that by another way in our project.